docs/solana/02-deployment-addresses.md

```diff
@@ -4,18 +4,20 @@ sidebar_position: 2
 title: "Deployment Addresses"
 ---
 
-This page contains the deployment addresses for the first SolSab release.
+This page contains the deployment addresses for SolSab.
 
 :::important
 
-Unlike EVM, the programs here adhere to the best practices of Solana development which includes upgradeability.
+SolSab programs adhere to the best practices for Solana development, which includes being upgradeable.
 
 :::
 
 ## Deployment Addresses
 
-All programs are deployed using
-[Anchor's verifiable builds](https://www.anchor-lang.com/docs/references/verifiable-builds) for deterministic bytecode.
+All programs have been built and deployed [verifiably](https://www.anchor-lang.com/docs/references/verifiable-builds),
+generating deterministic bytecode.
+
+### v0.1:
```

Comment:
Why not `v1.0`? Also, there is no need for `:` given its already marked as a heading.

Reply:
I don't think it would be confusing but if you two disagree then lets go with v0.1.

Technical versioning apart, its always good to see a protocol being released as `v1.0`. The onchain programs don't show any versions at all so I didn't think that it could be a problem.

On a side note, why even show `v0.1` on this page? We don't have any other version right now so may be we can skip it and keep it simple. Wdyt?

Reply:
I also think v0.1 is fine.

It's used for beta versions, yes, but this is our first ever Solana release. Nothing wrong with treating it like that.